gno.land/cmd/gnoland/start.go

```diff
@@ -372,10 +372,10 @@ func generateGenesisFile(genesisFile string, pk crypto.PubKey, c *startCfg) erro
 	gen.ConsensusParams = abci.ConsensusParams{
 		Block: &abci.BlockParams{
 			// TODO: update limits.
-			MaxTxBytes:   1_000_000,   // 1MB,
-			MaxDataBytes: 2_000_000,   // 2MB,
-			MaxGas:       100_000_000, // 100M gas
-			TimeIotaMS:   100,         // 100ms
+			MaxTxBytes:   1_000_000,     // 1MB,
+			MaxDataBytes: 2_000_000,     // 2MB,
+			MaxGas:       3_000_000_000, // 3B gas
```

Comment:
```suggestion
			MaxGas:       10_000_000_000, // 10B gas
```

Reply:
@moul @thehowlIn In the new gas configuration, one unit of gas is approximately equal to 1 nanosecond of execution time. Allowing 10 billion gas per block means permitting 10 seconds of execution time per block, which is too large. The current commit time is configured as 5 seconds. In a network, 3 seconds per block is a safer duration for a validator to join the network and catch up with the latest block playback.